C2CRIBuildDir/projects/C2C-RI/src/RIGUI/log4j2.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration packages="org.fhwa.c2cri.logger" status="TRACE" monitorInterval="30" dest="log4j_status.log">
+	<Appenders>
+		<RollingFile name="C2CRIOutput" fileName="C2CRIOutput.log" append="false" filePattern="C2CRIOutput-%i.dat">
+			<RIXMLLayout />
```

Comment:
Difference between <RIXMLLayout> and org.apache.log4j.xml.XMLLayout?

Reply:
Do we want a datetime on the .dat file?

Reply:
The RIXMLLayout writes xml that is formatted in the same way as the org.apache.log4j.xml.XMLLayout as far as I can tell. One reason for keeping the RIXMLLayout is that the apache one is dependent on at least 4 more JARs and in my local environment I was not able to get it to work. I'm guessing with more investigation we could get it to work but there is exceptional formatting that is used in some cases so I think we need to keep the RIXMLLayout.

Reply:
We certainly can put a datetime on the .dat file, although it is probably not necessary since the log file gets overwritten each time the application is opened. I think it should also be a .log file, that is a mistake on my part.